.github/workflows/publish-gem.yml

```
@@ -0,0 +1,30 @@
+name: Ruby Gem
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
```

```suggestion
    - name: Set up Ruby
```

We can skip version number if its using the .ruby-version file